package exception;

public class InvalidAmountException extends BankException {
    private static final long serialVersionUID = 1L;

    public InvalidAmountException(long amount) {
        super("Invalid amount: Rs." + amount + " (must be greater than 0)");
    }
}
